nilearn/plotting/surf_plotting.py

```diff
@@ -270,6 +277,24 @@ def plot_surf(surf_mesh, surf_map=None, bg_map=None,
             surf_map_faces = np.mean(surf_map_data[faces], axis=1)
         elif avg_method == 'median':
             surf_map_faces = np.median(surf_map_data[faces], axis=1)
+        elif avg_method == 'min':
+            surf_map_faces = np.min(surf_map_data[faces], axis=1)
+        elif avg_method == 'max':
+            surf_map_faces = np.max(surf_map_data[faces], axis=1)
+        elif callable(avg_method):
+            surf_map_faces = np.apply_along_axis(avg_method, 1, surf_map_data[faces])
+
+            ## check that surf_map_faces has the same length as face_colors
+            if surf_map_faces.shape != (face_colors.shape[0],):
+                raise ValueError('Array computed with the custom function from avg_method does not have the correct shape: {} != {}'.format(surf_map_faces.shape[0], face_colors.shape[0])
```

Comment:
beware the 80 char limit per line.

Reply:
My last commit (42cc8fb) fixes this, thanks!